services/DataCollectorImpl.groovy

```diff
@@ -42,7 +37,7 @@ class DataCollectorImpl extends DataCollector {
     }
 
     private void assembleResults(String file) {
-        String path = "output/files/${project.getName()}/${mergeCommit.getSHA()}/${file}/"
+        String path = this.outputPath + "/files/${project.getName()}/${mergeCommit.getSHA()}/${file}/"
```

Comment:
É só uma questão de estilo, mas eu particularmente acho mais legível quando usada a sintaxe do Groovy pra strings `"${this.outputPath}/files/${project.getName()}/${mergeCommit.getSHA()}/${file}/"`. Se você discordar disso, tudo bem.